src/olympia/zadmin/tasks.py

```diff
@@ -249,7 +251,17 @@ def fetch_langpack(url, xpi, **kw):
             # Not `version.files.update`, because we need to trigger save
             # hooks.
             file_.update(status=amo.STATUS_PUBLIC)
-        sign_file(file_)
+
+        # Create a dummy request to be able to tie the autograph enabling
+        # flag to our language pack users to enable signing for
+        # language packs
+        dummy_request = RequestFactory().get('/')
```

Comment:
😟 using test classes in live code makes me worry. If we're just using it below (and as far as I can see, we are) then why not just pass `use_autograph=True` or `use_autograph=False` as you think appropriate.

Or `autograph_flag = waffle.models.Flag.get('activate-autograph-signing)` then do `autograph.everyone` to get the default if you don't want to hard-code it? (or something - not tested)